.gitignore

```diff
@@ -5,3 +5,5 @@ tests/__pycache__
 .cache
 .vagrant
```

Comment:
As this role is not tested using vagrant anymore, I think we could remove `.vagrant` entry

Reply:
Done!